### PR TITLE
[webview_flutter] fix warning

### PR DESCRIPTION
**Pull request by rocketnik**
When the webview_flutter plugin compiles for iOS, it produces the following warning two times:
"'requiresuseractionformediaplayback' is deprecated: first deprecated in ios 10.0"

This PR disables those warnings in the same way, that other warnings of the same kind are already disabled.
The code leading to the warnings is fine, because it is there to explicitly handle the deprecations. Now that they are handled, the warnings can be ignored.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by stuartmorgan** [comment]
Thanks for the submission!

In the future, please do not delete the checklist that is in the PR template; it is there for a reason. This PR is missing required elements described in the checklist (I’ve restored it to the PR description), which need to be addressed before it moves forward with review.

I am marking the PR as a Draft. Please review the checklist, updating the PR as appropriate, and when that’s complete please feel free to mark the PR as ready for review.

**Review by cyanglaz — CHANGES_REQUESTED** [review]

**Comment by rocketnik** [comment]
Thank you for the review, I made appropriate fixes.

File: packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m [hunk 1]
```diff
@@ -572,10 +572,10 @@ - (void)updateAutoMediaPlaybackPolicy:(NSNumber *)policy
       if (@available(iOS 10.0, *)) {
         configuration.mediaTypesRequiringUserActionForPlayback = WKAudiovisualMediaTypeAll;
       } else if (@available(iOS 9.0, *)) {
-        configuration.requiresUserActionForMediaPlayback = true;
-      } else {
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
+        configuration.requiresUserActionForMediaPlayback = true;
```
[the remark this answers is not in the document]
Comment (rocketnik) [reply packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m:577]: fixed

File: packages/webview_flutter/webview_flutter_wkwebview/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+
```
[the remark this answers is not in the document]
Comment (rocketnik) [reply packages/webview_flutter/webview_flutter_wkwebview/CHANGELOG.md]: fixed

File: packages/webview_flutter/webview_flutter_wkwebview/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+## 2.7.3
+
+* Removes two compile time warnings
```
[the remark this answers is not in the document]
Comment (rocketnik) [reply packages/webview_flutter/webview_flutter_wkwebview/CHANGELOG.md]: fixed

File: packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m [hunk 4]
```diff
@@ -584,7 +587,10 @@ - (void)updateAutoMediaPlaybackPolicy:(NSNumber *)policy
       if (@available(iOS 10.0, *)) {
         configuration.mediaTypesRequiringUserActionForPlayback = WKAudiovisualMediaTypeNone;
       } else if (@available(iOS 9.0, *)) {
```
Comment (stuartmorgan) [inline packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m:589]: This was missed during a previous cleanup round apparently; we only support iOS 9 now, so this check isn't necessary, nor is the `else` block below. This should just become `else`, and the last block removed entirely.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m (hunk 4 above)
Comment (rocketnik) [reply packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m:589]: Hello Stuart, am I getting this right, that with only supporting iOS 9 now you mean, that iOS 9 is the lowest supported version? I make the appropriate changes.

Comment (rocketnik) [reply packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m]: fixed.

**Review by cyanglaz — APPROVED** [review]
LGTM

File: packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m (hunk 4 above)
Comment (stuartmorgan) [reply packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m]: Correct, sorry for the bad wording there.

**Review by stuartmorgan — APPROVED** [review]
LGTM, thanks!

**Comment by Hixie** [comment]
test-exempt: fixes warnings

**Comment by rocketnik** [comment]
Thank you for the great reviews!

**Pull request merged**